apps/rush-lib/src/schemas/command-line.schema.json

```diff
@@ -72,6 +72,11 @@
               "title": "Ignore Missing Script",
               "description": "Normally Rush requires that each project's package.json has a \"scripts\" entry matching the custom command name. To disable this check, set \"ignoreMissingScript\" to true.",
               "type": "boolean"
+            },
+            "ignoreDependencies": {
+              "title": "Ignore dependency relation between projects and just run everything in parallel",
```

Comment:
The title should be "Ignore dependencies".  (Frankly I'm not sure we need these titles at all, but we should maintain consistency.)

Reply:
This needs to be removed now.